packages/flutter/lib/src/rendering/error.dart

```diff
@@ -9,6 +9,7 @@ import 'object.dart';
 
 const double _kMaxWidth = 100000.0;
 const double _kMaxHeight = 100000.0;
+const String line = '\n──────────────────────\n\n';
```

Comment:
Looks still legible in all cases though printing multiple times overflows the parent container